packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -251,10 +251,10 @@ export abstract class PersistentStream<
   markIdle(): void {
     this.idle = true;
     this.queue
-      .schedule(() => {
+      .scheduleWithDelay(() => {
         return this.handleIdleCloseTimer();
```

Comment:
`handleIdleCloseTimer` has its own logic to prevent itself from running. Do you think you could migrate this code to use this new functionality. We already cancel the timer in Android: https://cs.corp.google.com/piper///depot/google3/java/com/google/android/gmscore/dev/thick_client/firebase-firestore/src/com/google/firebase/firestore/remote/AbstractStream.java?rcl=184463491&l=362